### PR TITLE
Get cmake generator from environment in windows for lockfile example

### PR DESCRIPTION
**Pull request by czoido**

### CODE REVIEW

**Review by jgsogo — APPROVED** [review]

**Review by jgsogo — CHANGES_REQUESTED** [review]
Wait! Maybe we should use a default value, or do something that lets the user know what is happening

**Comment by memsharded** [comment]
> I think we should get the value, check it exists, and if it doesn't we can write a message to the user: populate this environment variable with your generator before calling this example... or add only the -G xxxxx if the variable exists (I don't know if CMake will choose an existing VS generator by default)

Yes, the -G cmake generator is generally necessary in Windows. It is necessary to add something in the script, that will fail if not defined and force to define it, otherwise it is very confusing for the user.

**Review by jgsogo — APPROVED** [review]
🙌  Really cool to see examples green again!

Thanks!

**Review by Featzaza999 — COMMENTED** [review]
cmake $ GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE = $ BUILD_TYPE

File: .ci/run.py [hunk 1]
```diff
@@ -54,14 +54,21 @@ def get_examples_to_skip(current_version):
     skip = []
     # Given the Conan version, some examples are skipped
     required_conan = {
+        version.parse("1.28.2"): [
+            './libraries/dear-imgui/basic', # solved bug for system packages and components
+            ],
+        version.parse("1.28.0"): [
+            './features/lockfiles/intro', # new lockfiles version
+            './features/lockfiles/ci', # new lockfiles version
+            './features/lockfiles/build_order', # new lockfiles version
```
Comment (Featzaza999) [inline .ci/run.py:63]: ``

**Pull request merged**